packages/devextreme/js/__internal/core/widget/widget.ts

```diff
@@ -265,8 +266,9 @@ class Widget<
     return this._getActiveElement();
   }
 
-  _isFocusTarget(element: Element): boolean {
+  _isFocusTarget(element: Element | undefined): boolean {
     const focusTargets = $(this._focusTarget()).toArray();
+    // @ts-expect-error ts-error
     return focusTargets.includes(element);
```

Comment:
Let's fix error instead of muting
```suggestion
    return !!element && focusTargets.includes(element);
```

Reply:
I'll consider in the further PRs